src/foundation.cs

```diff
@@ -13612,30 +13692,30 @@ partial interface NSAttributedString {
 		nint UnderlineByWordMaskAttributeName { get; }
 
 #if MONOMAC
-		[Mac (10,15)]
 		[Field ("NSTextScalingDocumentAttribute", "AppKit")]
 #else
-		[iOS (13,0), TV (13,0), Watch (6,0)]
 		[Field ("NSTextScalingDocumentAttribute", "UIKit")]
 #endif
+		[Mac (10,15)]
+		[iOS (13,0), TV (13,0), Watch (6,0)]
 		NSString TextScalingDocumentAttribute { get; }
 
 #if MONOMAC
-		[Mac (10,15)]
 		[Field ("NSSourceTextScalingDocumentAttribute", "AppKit")]
 #else
-		[iOS (13,0), TV (13,0), Watch (6,0)]
 		[Field ("NSSourceTextScalingDocumentAttribute", "UIKit")]
 #endif
+		[Mac (10,15)]
+		[iOS (13,0), TV (13,0), Watch (6,0)]
 		NSString SourceTextScalingDocumentAttribute { get; }
 
 #if MONOMAC
```

Comment:
Maybe a conditionally defined `const string Library = "AppKit"` or `"UIKit"` could be used here, but I guess not in the scope of this PR.

Reply:
Not a bad idea, I was just keeping this mechanical enough to keep moving, since we have to touch nearly every single file.